docs/LwM2M_Client/OpenThread/Adding_device.md

```diff
@@ -0,0 +1,145 @@
+# Adding nRF board to OpenThread
+
+Integrate your Nordic nRF52840-DK board to an active OpenThread Border Router.
+
+## Prerequisites
+
+- An active **OpenThread Border Router**.
+- The **nRF52840-DK** board with a USB cable.
+- Installed **minicom** or **RealTerm** (for Linux or Mac) or **PuTTy** (for Windows) or other serial communication program.
+- Installed **nrfjprog** from [Nordic Semiconductor page](https://www.nordicsemi.com/Products/Development-tools/nrf-command-line-tools/download)
+- A user with access to the [**Coiote IoT Device Management**](https://eu.iot.avsystem.cloud/ui/device/inventory) platform and appropriate permissions.
+
+## Connecting to the OpenThread
+
+Check your active Border Router **OTBR_IP_address** and **OTBR_port** on which your Border Router works. This will be necessary for further steps.
+
+### Connection through Wi-Fi
+
+You have to be connected to the same Wifi network as your OpenThread Border Router. To connect your device to Border Router through Wifi, you should go to `OTBR_IP_address`.
+
+!!! Note
+    There is an option to connect to your Border Router through ssh by `ssh -L OTBR_port:OTBR_IP_addres:80 -L 8081:OTBR_hostname` command, then go to: `http://OTBR_IP_address:80`. The additional forwarding for port 8081 is necessary for the ***Topology*** view to work. Also, port 80 is necessary due to the HTTP protocol.
```

Comment:
I'm not sure what you wanted to to say here. The first part of the part of the `ssh` command `-L OTBR_port:OTBR_IP_addres:80` means port forwarding. You forward your OTBR_port (on your local machine) to OTBR_IP_addres:80 but your not connecting to anything with the ssh command so I guess you won't forward anything. We are using port forwarding to connect to our OTBR but this may not be the case for someone outside our team.